go.mod

```diff
@@ -2,6 +2,11 @@ module github.com/rancher/steve
 
 go 1.20
 
+replace (
+	github.com/rancher/apiserver v0.0.0-20230831052300-120e615b17ba => github.com/KevinJoiner/rancher-apiserver v0.0.0-20240111151544-0ab61a41a7fa
+	github.com/rancher/dynamiclistener v0.3.6-rc2.0.20230831052350-0132d96ec2c5 => github.com/KevinJoiner/rancher-dynamiclistener v0.3.4-0.20240111212255-e6451ba1e8e0
```

Comment:
Remove and update version once https://github.com/rancher/dynamiclistener/pull/88 is merged